templates/pgb_config.j2

```diff
@@ -0,0 +1,31 @@
+[databases]
+{% for name, database in databases.items() %}
+{{ name }} = host={{ database.host }} dbname={{ database.dbname }} port={{ database.port }} auth_user={{ database.auth_user }}
+{% endfor %}
+
+[pgbouncer]
+listen_addr = *
+listen_port = {{ listen_port }}
+logfile = {{ log_file }}
+pidfile = {{ pid_file }}
+stats_users = {{ stats_user }}
+auth_type = md5
+user = postgres
+max_client_conn = 10000
+ignore_startup_parameters = extra_float_digits,options
+server_tls_sslmode = prefer
+so_reuseport = 1
+unix_socket_dir = {{ socket_dir }}
+pool_mode = {{ pool_mode }}
+max_db_connections = {{ max_db_connections }}
+default_pool_size = 4
+min_pool_size = 2
+reserve_pool_size = 2
```

Comment:
Aren't those values [dynamic](https://github.com/canonical/pgbouncer-k8s-operator/blob/0220e2129d617f2311f5a649da8f9066692bf75c/lib/charms/pgbouncer_k8s/v0/pgb.py#L353)?

Reply:
Good catch. Should be calculated now.